rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,572 @@
+	echo "      hist                Trace the determination of which shell history file to use"
+	echo "      terminal            Trace the terminal color mode detection"
+	echo "    You can specify multiple modes, separated by commas, e.g. --trace=custom,hist"
+	echo ""
+}
+
+function options_to_env() {
+	local kv
+	local k
+	local v
+
+	for option in ${options[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)